ambry-api/src/main/java/com/github/ambry/accountstats/AccountStatsStore.java

```diff
@@ -31,6 +32,14 @@ public interface AccountStatsStore {
    */
   void storeHostAccountStorageStats(HostAccountStorageStatsWrapper statsWrapper) throws Exception;
 
+  /**
+   * Delete {@link HostAccountStorageStats} for the given host.
+   * @param hostname The hostname
+   * @param port The port
+   * @throws Exception
+   */
+  void deleteHostAccountStorageStatsForHost(String hostname, int port) throws Exception;
```

Comment:
Minor. Can method name be `deleteHostAccountStorageStats`?

Reply:
I think the method name is fine. HostAccountStorageStats is a term we use to represent the account stats reported by host. We have other method that delete this for cluster, for container, account etc...

Reply:
Sure.